rpm/packagescanner.go

```diff
@@ -169,17 +169,24 @@ func (ps *Scanner) ScanContext(ctx context.Context, layer *claircore.Layer) ([]*
 	// permissions later. Otherwise, some layers will have directories without
 	// the write bit created and the permissions set before their contents are
 	// created.
-	tarcmd := exec.CommandContext(ctx, "tar", "-xC", root)
+	errbuf := bytes.Buffer{}
+	tarcmd := exec.CommandContext(ctx, "tar", "-xC", root, "--exclude", "dev")
```

Comment:
@hdonnay a little bit of nit, but a comment here about how unprivileged containers will fail without this would help readers understand wh we now exclude.

Reply:
Agree